protos/flyteidl/event/event.proto

```diff
@@ -87,6 +87,12 @@ message NodeExecutionEvent {
 
     // Friendly readable name for the node
     string node_name = 13;
+
+    // Whether this node launched a subworkflow.
```

Comment:
Is this a subworkflow or a launchplan? or more?

Reply:
just a subworkflow